stacker/session_cache.py

```diff
@@ -15,13 +25,12 @@ def get_session(region):
         :class:`boto3.session.Session`: A boto3 session with
             credential caching
     """
-    session = botocore.session.get_session()
-    if region is not None:
-        session.set_config_variable('region',  region)
-    c = session.get_component('credential_provider')
+    session = boto3.Session(region_name=region, profile_name=profile)
```

Comment:
I think the reason we had to use botocore originally was because boto3 didn't have this ability at the time.  Do we need to update our dependency for boto3 to a specific minimum version for this?

Reply:
We lock on "boto3>=1.3.1", [which has support for both of these params](https://github.com/boto/boto3/blob/1.3.1/boto3/session.py#L48-L50), so we should be ok.